src/cargo/ops/registry.rs

```diff
@@ -245,6 +245,20 @@ pub fn publish(ws: &Workspace<'_>, opts: &PublishOpts<'_>) -> CargoResult<()> {
         }
     }
 
+    // Print success note on publish
+    if opts.registry.is_none() {
+        if let Some(ref registries) = pkg.publish() {
+            if registries[0] == CRATES_IO_REGISTRY {
+                if let Ok(packages) = opts.to_publish.get_packages(ws) {
+                    for package in packages {
+                        opts.config
+						.shell().note(format!("Your crate was successfully published to crates.io! You can visit it at https://crates.io/crates/{} .", package.name()))?
+                    }
+                }
+            }
+        }
+    }
+
```

Comment:
> The current cargo publish output isn't very helpful or gives a lot of feedback.
>
> It's something like:
> And that's it, it ends with a boring "Updating index" message. I

The message you showed I'm assuming is from a stable release.  As you saw in the test, `master` already ends with
```
    Updating crates.io index
   Packaging delay v0.0.2 (/Users/eric/Proj/rust/cargo/target/tmp/cit/t0/foo)
    Packaged 3 files, 761.0B (569.0B compressed)
   Uploading delay v0.0.2 (/Users/eric/Proj/rust/cargo/target/tmp/cit/t0/foo)
    Uploaded delay v0.0.2 (/Users/eric/Proj/rust/cargo/target/tmp/cit/t0/foo)
note: Waiting for `delay@0.0.2` to be available at registry `crates-io`.
You may press ctrl-c to skip waiting; the crate should be available shortly.
     Waiting [===>                       ] 11/60
   Published delay v0.0.2 (/Users/eric/Proj/rust/cargo/target/tmp/cit/t0/foo) has been successfully published to registry `crates-io`
```
See #11713

Adding another success message feels a bit excessive in the amount of output we are showing the user.

Reply:
Yeah you're right. I guess I just have a weird / old version (I made the issue in February, maybe the stable ver. there has changed a lot?)

Anyways, I'm going to close the PR